modules/installation-configuration-parameters.adoc

```diff
@@ -627,6 +627,12 @@ endif::openshift-origin[]
 |Specify one or more repositories that may also contain the same images.
 |Array of strings
 
+ifdef::aws[]
+|`platform.aws.lbType`
+|Required to set the NLB load balancer type in AWS. Valid values are `Classic` or `NLB`. The default is Classic if no value is provided. The installation program sets the value provided here in the ingress cluster config object. If you do not specify a load balancer type for other ingress controllers, they will use the configuration set in this parameter.
```

Comment:
```suggestion
|Required to set the NLB load balancer type in AWS. Valid values are `Classic` or `NLB`. If no value is specified, the installation program defaults to `Classic`. The installation program sets the value provided here in the ingress cluster configuration object. If you do not specify a load balancer type for other ingress controllers, they use the type set in this parameter.
```

- See the comment above on API object formatting. 
- The ISG recommends [avoiding future tense](https://www.ibm.com/docs/en/ibm-style?topic=grammar-verbs#tense).
- See the [ISG guidance](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#config) on config. _config map_ and _build config_ are listed as exceptions in the [SSG](https://redhat-documentation.github.io/supplementary-style-guide/).
- The suggestion to use _type_ instead of _configuration_ in the last sentence is optional. It just sounds a little better to my ear.